### ISSUE TITLE
github.com/motemen/gore/commands.go:246: not enough arguments in call to mainScope.LookupParent

### ISSUE THREAD

**Issue by slyphon** [motemen/gore#22]
I'm kind of new to Go, was attempting to install gore and ran into this:

```
$ mkdir /tmp/gore
$ cd /tmp/gore
$ ls
$ export GOPATH=$(pwd -P)
$ go get github.com/motemen/gore
# github.com/motemen/gore
/private/tmp/gore/src/github.com/motemen/gore/commands.go:246: not enough arguments in call to mainScope.LookupParent
$ go env
GOARCH="amd64"
GOBIN=""
GOCHAR="6"
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/private/tmp/gore"
GORACE=""
GOROOT="/opt/twitter/Cellar/go/1.4.2/libexec"
GOTOOLDIR="/opt/twitter/Cellar/go/1.4.2/libexec/pkg/tool/darwin_amd64"
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fno-common"
CXX="clang++"
CGO_ENABLED="1"
$ go version
go version go1.4.2 darwin/amd64
```

I've tried going back roughly 10 versions of master, and each one fails with the same message. Is this a known issue? Am I missing something obvious? Is there any other information I can provide which would help?

**Comment by syohex** [comment]
https://go.googlesource.com/tools/+/665374f1c86631cf73f4729095d4cf4313670545

This change causes this issue. It changes `Scope.LookupParent` signature.

**Comment by motemen** [comment]
Thank you for reporting, it was because API changes on golang.org/x/tools as @syohex says. I have merged his PR #23, so you should now be able to compile gore.

**Comment by slyphon** [comment]
thank you so much! I'm glad it wasn't just me being dumb :)
